docs/source/tutorials/mantid_basic_course/algorithms_workspaces_and_histories/04_algorith_histories.rst

```diff
@@ -16,20 +16,23 @@ the GUI.
    open up the Algorithm History window. In the list of Algorithms, 
    click on Rebin v.1 and you should see the following:
 
-   |ThisFilenameislying|
-   
-   This reveals the history done to this workspace, i.e. Load to load
-   the workspace, SumSpectra, and Rebin.
-   To replay the history do:
+.. The image file name is lying!
```

Comment:
That is this line?

Reply:
I didn't change the image file name, which was lying about what workspace it related to, now I have and this is removed.